File: BusinessCore/Domain/Payment.cs
namespace BusinessCore.Domain
{
    public class Payment
    {
        public int CardNo { get; set; }
        public string CardHolder { get; set; }
        public int AmountTrxn { get; set; }
        public int CurrencyCode { get; set; }
        public int ProcessingCode { get; set; }
        public int SystemTraceNr { get; set; }
        public int FunctionCode { get; set; }
    }
}

File: angular/src/app/core/models/payment.ts

export interface Payment {
    cardNo?: any;
    cardHolder?: any;
    amountTrxn?: any;
    currencyCode?: any;
    processingCode?: any;
    systemTraceNr?: any
    functionCode?: any
}

export interface PaymentResponse {
    responseCode: any
    message: any
    approvalCode: any
    dateTime: any
}